dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessService.java

```diff
@@ -2194,6 +2204,7 @@ public int switchVersion(ProcessDefinition processDefinition, ProcessDefinitionL
         return result;
     }
 
+    @Transactional(rollbackFor = RuntimeException.class)
```

Comment:
There is no exception thrown in the code. I think adding this here will not work.

Reply:
this code will not throw check exception, but cloud throw runtime exception while database or network has problem.